src/transformStateWithClones.js

```diff
@@ -7,7 +7,38 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const addOption = 'addProperties';
+  const removeOption = 'removeProperties';
+  const clearOption = 'clear';
```

Comment:
same here with constants naming
```suggestion
  const ADD_OPTION = 'addProperties';
  //...
```